processFlows.json

```diff
@@ -558,6 +549,747 @@
       },
       { "op": "And" },
       { "op": "Or" },
+      { "op": "And" },
+      {
+        "restriction": {
+          "OutputHasMetadata": {
+            "index": 0,
```

Comment:
`index: 2`? If this is for the new `MATCH2` token?

Reply:
if you look this is adding `originalId` prevention to match2-create. Should have been there before really

Reply:
yep you're right